packages/envision-docs/src/pages/components/cards.md

```diff
@@ -0,0 +1,357 @@
+---
+title: Cards
+---
+
+<span class="env-badge env-badge--info">Added in 9.0.1</span>
+
+## Cards & Cardholders
+
+Use Cards for displaying information about for example a page, a group or a user.
+
+A Card is a simple container that may have a header and a footer. A Card does not have
+it's own colors, so combine with `env-block` classes to set colors and `env-shadow` to set box shadow.
```

Comment:
Tror man kan bryta upp denna mening i två. Något så här ish?

_A Card has no colors or shadows of its own. To add colors and shadows, you may instead combine the "env-block" and "env-shadow" classes._